mac/Focalboard/ViewController.swift

```diff
@@ -219,5 +261,13 @@ class ViewController:
 	@IBAction func navigateToHome(_ sender: NSObject) {
 		loadHomepage()
 	}
+
+	func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
+		guard let body = message.body as? [String: String], let type = body["type"], let blob = body["settingsBlob"] else {
+			NSLog("Received unexpected script message \(message.body)")
+			return
+		}
+		NSLog("Received script message \(type): \(Data(base64Encoded: blob).flatMap { String(data: $0, encoding: .utf8) } ?? blob)")
+	}
```

Comment:
not really sure what does this function does other than receiving a message and printing it. Am I missign something?

Reply:
No, that's precisely it. 😅 

I had mentioned this above in the PR description:

> The WebKit messaging is only needed for logging. Unfortunately, it bloats the code a little so we could also remove it maybe. Here's an example of log messages on start-up / shutdown:

I found this helpful during development but it does add some complexity so we can take it out if there are concerns. Alternatively, we could also wrap it in something like `#if DEBUG ... #endif` so that we still have it available for future development but it doesn't land in the release build.